meilidb-schema/src/lib.rs

```diff
@@ -145,7 +145,7 @@ struct InnerSchema {
 }
 
 impl Schema {
-    fn to_builder(&self) -> SchemaBuilder {
+    pub fn to_builder(&self) -> SchemaBuilder {
```

Comment:
Do not make this public if you can already use the `Serialize` impl to format it like you expect.